containers/cuda/ilab-cuda.alias

```diff
@@ -0,0 +1,22 @@
+# shellcheck shell=bash
+
+ilab-cuda() {
+    declare INSTRUCTLAB_IMAGE=${INSTRUCTLAB_IMAGE:-localhost/instructlab:cuda}
+    declare GPU_INDEX=${GPU_INDEX:-0}
+    declare ILABDIR="${HOME}/.config/instructlab"
+
+    mkdir -p "${ILABDIR}"
+    podman unshare chown 1001:1001 -R "${ILABDIR}"
+
+    if [ ! -f "${ILABDIR}/config.yaml" ] && [ "$1" != "init" ]; then
+        echo "Error: \`config.yaml\` does not exists, please run \`ilab init\` or point to a valid configuration file using \`--config=<path>\`."
```

Comment:
```suggestion
        echo "Error: \`config.yaml\` does not exist, please run \`ilab init\` or point to a valid configuration file using \`--config=<path>\`."
```

Reply:
The alias file was originally written by Fabien. I could drop it from my PR and ask Fabien to submit it in a separate PR when is is available.

Reply:
I have removed the alias file from the PR and its documentation. We can add it later.